lars-reimann/api-editor
# fix: incorrect handling of properties

Closes #309.

### Summary of Changes

* Fix incorrect handling of properties
* Add `@getter`, `@setter`, or `@deleter` to the names of the corresponding functions to uniquely identify them

### Screenshots (if necessary)

![image](https://user-images.githubusercontent.com/2501322/143763627-8e039552-fea9-4605-84f8-7ada23db674c.png)

### Testing instructions

1. Import the new scikit-learn data.
2. Check that the [`power`](http://localhost:3000/#/sklearn/sklearn._loss.glm_distribution/TweedieDistribution/power@getter) property now works properly. This is the only property we have in the data.
